dev-support/hbase-personality.sh

```diff
@@ -660,9 +660,9 @@ function hadoopcheck_rebuild
   else
     yetus_info "Setting Hadoop 3 versions to test based on branch-2.4+/master/feature branch rules"
     if [[ "${QUICK_HADOOPCHECK}" == "true" ]]; then
-      hbase_hadoop3_versions="3.1.2 3.2.2 3.3.1"
+      hbase_hadoop3_versions="3.1.4 3.2.3 3.3.1"
```

Comment:
3.3.1 should be 3.3.2

Reply:
@Apache9 I removed 3.3.2 because it's currently broken, see [HBASE-26954](https://issues.apache.org/jira/browse/HBASE-26954). If we restore it here, all PRs and nightlies will fail.